### PR TITLE
fix: fetching operator schema from backend

### PR DESCRIPTION
**Pull request by gurjmatharu**
https://github.com/bcgov/cas-cif/issues/1475

### CODE REVIEW

**Review by tmastrom — COMMENTED** [review]
I would recommend removing `"default": "undefined"` from the json schema since the strings are showing up in the frontend when adding a new contact 
Screenshot 2023-03-27 at 3 42 44 PM

File: app/components/Operator/OperatorForm.tsx [hunk 1]
```diff
@@ -1,19 +1,125 @@
...
+        </h2>
+        <SavingIndicator
+          isSaved={!isUpdatingFormChange && !isCommittingFormChange}
+        />
+      </header>
+      <FormBase
+        {...props}
+        id="operator-form"
+        formData={newFormData}
+        schema={props.schema}
```
Comment (tmastrom) [inline app/components/Operator/OperatorForm.tsx:105]: why don't we pass a "uiSchema" prop?

Comment (gurjmatharu) [reply app/components/Operator/OperatorForm.tsx:105]: not needed for operator, as none of the forms have any unique ui

**Review by tmastrom — APPROVED** [review]
LGTM

**Comment by pbastia** [comment]
The "Changes Saved" indicator moved under the header instead of in-line on the right

**Pull request merged**